extensions/interactions/interaction-attributes-extractor.service.spec.ts

```diff
@@ -39,25 +38,17 @@ describe('Interaction attributes extractor service', () => {
 
   it('should properly extract customization arguments values from attributes',
     () => {
-      const choicesWithValue = hes.objToEscapedJson([{
-        content_id: 'ca_placeholder_0',
-        html: 'Enter here.'
-      }]);
-      const allowMultipleItemsInSamePositionWithValue = hes.objToEscapedJson(
-        true
-      );
+      const recommendedExplorationIdsWithValue = hes.objToEscapedJson(['0']);
```

Comment:
Why are we modifying this test?

Reply:
This test is for interactions, that are not still migrated, we use `DragAndDropSortInput` for example. But now `DragAndDropSortInput` is migrated and we use endExploration as example. 

we decided to merge EndExploration in codebase at last.